cadasta_test_suites.py

```diff
@@ -115,16 +115,26 @@
 
 # Get all tests from accounts test classes
 new_registration = unittest.TestLoader().loadTestsFromTestCase(NewRegistration)
-registration_attempt_existing_username = unittest.TestLoader().loadTestsFromTestCase(RegistrationAttemptUsernameNotAvailable)
```

Comment:
> Isn't there a way to discover tests automatically?

As I understand it, the test suite currently uses the basic Python unittest framework which AFAIK needs this boilerplate code. I have been able to run the tests using the pytest framework which makes this file (cadasta_test_suites.py) unnecessary because pytest auto-discovers tests. But we also need to rename the test files to either have test_*.py or *_test.py as their filenames to make things fully discoverable. I haven't updated the suite to use pytest for this PR.